Portfolio.css

```diff
@@ -0,0 +1,69 @@
+* {
```

Comment:
Please remove this file if it's not being used (https://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-a-git-repository)